crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -320,13 +321,26 @@ impl RefreshTask {
         let dataset_name = self.dataset_name.clone();
         let filter_converter = self.get_filter_converter(refresh);
 
+        let component_type = if self.is_view_acceleration() {
```

Comment:
Instead of calculating this based on a bool, we should have a method that returns the `component_type` directly.

Reply:
Updated, thank you - minimized changes to simplify review. We can have next iteration to refactor/rename naming w/o changing functionality next.